web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -244,14 +246,14 @@ export function initRepoBranchTagSelector(selector) {
 export default sfc; // activate IDE's Vue plugin
 </script>
 <template>
-  <div class="ui dropdown custom">
-    <button class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-m-0" @click="menuVisible = !menuVisible" @keyup.enter="menuVisible = !menuVisible">
-      <span class="text gt-df gt-ac gt-mr-2">
+  <div class="ui dropdown custom gt-w-full gt-h-full">
+    <button type="button" class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-sb gt-m-0 gt-w-full gt-h-full" @click="menuVisible = !menuVisible" @keyup.enter="menuVisible = !menuVisible">
+      <span class="text gt-df gt-ac gt-mr-2 gt-ellipsis">
```

Comment:
TBH, I am not a fan of filling these fine-tuning helpers everywhere.

Personally I have no motivation to approve .... but I won't block.

Reply:
Then what's the point of having helper-classes?
Or do you have another idea to Do it better?

Reply:
Some opinions in my mind:

* What's the purpose of using CSS file and CSS classes? If the helpers should be used everywhere, then why there are CSS classes like ".my-button". Using one or two helpers to fine-tune something seems good, but using 5-7 helpers seems to be quite hacky IMO.
* Should it encourage to write reusable code? If the same helpers are written for more than one time, then why not introducing a clearly defined CSS class for it.

There is a long argument in the frontend development, some people really like using helpers as much as possible, but I really dislike abusing them.

Reply:
I see your point, but there are ups and downs to each approach.
Because its a vue-component I like the helper-classes more then normal css

Reply:
There are still new classes has prefix `gt-` introduced. Could you replace them with tailwind?

Reply:
And it's better to use plain javascript instead of jquery because we are removing jquery.

Reply:
I don't mind the old classes, it keeps the diff minimal and they will later be replaced in a find-replace action anyways. jQuery in this PR is also not really an issue as it would better be done in a whole-file rewrite.